deepmd/dpmodel/fitting/property_fitting.py

```diff
@@ -131,7 +130,7 @@ def serialize(self) -> dict:
             "type": "property",
             "task_dim": self.task_dim,
             "intensive": self.intensive,
-            "property_name": self.property_name,
+            "property_name": self.var_name,
```

Comment:
really need to serialize in the model ckeckpoint? the `var_name` has been serialized by its parent class

Reply:
I have modified it.
<img width="665" alt="image" src="https://github.com/user-attachments/assets/1d70c3cc-b4b2-48e0-9150-06e73da9b7b9" />